src/Language/Haskell/GHC/ExactPrint/Preprocess.hs

```diff
@@ -185,7 +189,8 @@ getPreprocessedSrcDirectPrim cppOptions src_fn = do
   hsc_env <- GHC.getSession
   let dfs = GHC.extractDynFlags hsc_env
       new_env = GHC.replaceDynFlags hsc_env (injectCppOptions cppOptions dfs)
-  (dflags', hspp_fn) <- GHC.liftIO $ GHC.preprocess new_env (src_fn, Nothing)
+  (dflags', hspp_fn) <-
+      GHC.liftIO $ GHC.preprocess new_env (src_fn, Just (GHC.Cpp GHC.HsSrcFile))
```

Comment:
In fact, it is not clear that this is the right flag - I think we want to run the whole preprocessor pipeline rather than starting from the CPP phase. It is true that 99% of preprocessing is by CPP but other preprocessing might also be necessary.